torch/fx/graph_module.py

```diff
@@ -310,6 +337,15 @@ def wrapped_call(self, *args, **kwargs):
                 sys.excepthook = old_excepthook
         cls.__call__ = wrapped_call
 
+    def __reduce_package__(self, exporter: PackageExporter):
+        module_id = exporter.get_unique_id()
+        exporter.save_source_string(f'fx-generated._{module_id}', self.code)
+        self._fx_generated_id = module_id
```

Comment:
This never gets deleted on the other end. Might make more sense to keep this as a separate argument. Only reason it is getting put in self here is because of trying to match the arguments to the other deserialize call.